code/datums/supplypacks/security.dm

```diff
@@ -31,7 +31,7 @@
 	cost = 40
 	containertype = /obj/structure/closet/crate/secure/gear
 	containername = "Armor crate"
-	access_armory //VOREStation Add - Armor is for the armory.
```

Comment:
This was not put in the correct file to begin with, much less coded correctly. This line should not be here at all.

Reply:
Should it stay deleted because VORE EDIT is no longer wanted? Because in _vr files, nowhere is there access restriction defined for that crate. Which means that addition was not done at all at any point, and it still used access_security.

Reply:
It should stay deleted because whoever added this line in the first place is a moron.

Not only did they not add the change to the correct file, they added a change that _literally doesn't do anything_. `access_armory` is just sitting there uselessly. I don't know how this even compiled.

Reply:
My question was. Whether the change in access on VR side should be kept at all or not. But that answers it and I will just move it to security_vr